Client/Base/frontend/components/basket/basket.component.scss

```diff
@@ -64,22 +70,33 @@
         a + a {
             margin-top: var(--button-gap);
         }
+
+        // Clear Cart Button
```

Comment:
write css class names not in camel case but with dash: clear-cart-confirmation-button